reference-architecture/azure-ansible/README.md

```diff
@@ -0,0 +1,75 @@
+# RedHat Openshift Enterprise cluster on Azure
+
+When creating the RedHat Openshift Enterprise Cluster on Azure, you will need a SSH RSA key for access.
+
+## SSH Key Generation
+
+1. Windows - https://www.digitalocean.com/community/tutorials/how-to-create-ssh-keys-with-putty-to-connect-to-a-vps
+2. Linux - https://help.ubuntu.com/community/SSH/OpenSSH/Keys#Generating_RSA_Keys
+3. Mac - https://help.github.com/articles/generating-ssh-keys/#platform-mac
+
+## Create the cluster
+### Create the cluster on the Azure Portal
+
+<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2Fopenshift%2Fopenshift-ansible-contrib%2Fmaster%2Fazure-ansible%2Fazuredeploy.json" target="_blank">
+    <img src="http://azuredeploy.net/deploybutton.png"/>
+</a>
+<a href="http://armviz.io/#/?load=https%3A%2F%2Fraw.githubusercontent.com%2Fopenshift%2Fopenshift-ansible-contrib%2Fmaster%2Fazure-ansible%2Fazuredeploy.json" target="_blank">
+    <img src="http://armviz.io/visualizebutton.png"/>
+</a>
+### Add Additional Nodes - Experimental
+<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2Fopenshift%2Fopenshift-ansible-contrib%2Fmaster%2Fazure-ansible%2Fazureexpand.json" target="_blank">
+    <img src="http://azuredeploy.net/deploybutton.png"/>
+</a>
+
+### Add Test Single Node
+Single VM Test Machine - Experimental
+<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2Fopenshift%2Fopenshift-ansible-contrib%2Fmaster%2Fazure-ansible%2Fonenode.json" target="_blank">
+    <img src="http://azuredeploy.net/deploybutton.png"/>
+</a>
+
+### Create the cluster with powershell
+
+```powershell
+New-AzureRmResourceGroupDeployment -Name <DeploymentName> -ResourceGroupName <RessourceGroupName> -TemplateUri https://raw.githubusercontent.com/openshift/openshift-ansible-contrib/master/reference-architecture/azure-ansible/azuredeploy.json
+```
+
+
+## Parameters
+SSHKeyData - Public Key - Copy/Past frome .ssh/id_rsa.pub - pbcopy < ~/.ssh/id_rsa.pub
+PoolId - From RREL - subscription-manager list --available
+SSHPrivate Data - Base64 Encoded id_rsa - cat ~/.ssh/id_rsa | base64 | pbcopy
```

Comment:
May be helpful: http://superuser.com/questions/288320/whats-like-osxs-pbcopy-for-linux